apps/client/src/common/api/settings.ts

```diff
@@ -26,3 +26,10 @@ export async function postSettings(data: Settings): Promise<AxiosResponse<Settin
 export async function postShowWelcomeDialog(show: boolean) {
   axios.post(`${settingsPath}/welcomedialog`, { show });
 }
+
+/**
+ * Allows setting offset mode
+ */
+export async function postOffsetMode(mode: OffsetMode) {
+  axios.post(`${settingsPath}/offsetmode`, { mode });
+}
```

Comment:
our (soft) rules indicate that this feature, which is not persisted, should be communicated over websockets